### PR TITLE
Test olympus and village

### PR DESCRIPTION
**Pull request by ninolutter**
Build village and Olympus with new layout,terrain and cloud effects

### CODE REVIEW

**Comment by KRademacher** [comment]
Changes in the GameReady folder?

File: Assets/_Scenes/Scene_OpenWorld/Prefabs/Fireflies/FireflyParticleCollision2.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FireflyParticleCollision2 : MonoBehaviour
+{
```
Comment (L5478) [inline Assets/_Scenes/Scene_OpenWorld/Prefabs/Fireflies/FireflyParticleCollision2.cs]: Are you using this script somewhere? cuz I couldn't find it. And if you are, who made this script? Do we have rights to use it?

File: Assets/_Scenes/Scene_OpenWorld/Prefabs/Fireflies/FireflyParticleAudio2.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FireflyParticleAudio2 : MonoBehaviour
+{
```
Comment (L5478) [inline Assets/_Scenes/Scene_OpenWorld/Prefabs/Fireflies/FireflyParticleAudio2.cs]: Are you using this script somewhere? cuz I couldn't find it. And if you are, who made this script? Do we have rights to use it?

**Comment by L5478** [comment]
> 
> 
> Changes in the GameReady folder?
I think those are new assets

**Pull request merged**